Gruntfile.js

```diff
@@ -259,7 +308,49 @@ module.exports = function (grunt) {
         grunt.task.run(tasks);
     });
 
-    grunt.registerTask("buildStylus", ["clean:stylus", "stylus"]);
+    grunt.registerMultiTask("distributions", "Enables a project to split its files into a set of modules. A module's information is stored in a json file containing a name for the module, the files it contains, and other modules it depends on. The module files can then be accumulated into various configurations of included and excluded modules, which can be fed into other plugins (e.g. grunt-contrib-concat) for packaging.", function () {
+        // Merge task-specific and/or target-specific options with these defaults.
+        var options = this.options({
+            name: this.target,
+            source: true,
+            target: "all",
+            compress: false
+        });
+
+        if (options.exclude || options.include) {
+            options.target = "custom";
+        }
+
+        setBuildSettings(options);
+
+        var concatTask = options.compress ? "uglify:" : "concat:";
+        var tasks = [
+            "clean:build",
```

Comment:
Thanks, @jobara - this pull looks generally good. Whilst we are touching the build scripts in this area, I thought it might be appropriate to fix up our clean:build strategy a little more:

Since we merged in https://github.com/fluid-project/infusion/pull/720 I feel we've gone from deleting too much build junk to deleting too little. We now accumulate an entire copy of the all the frameworks's JS source in build/ after a build, even though it is all included within one of the build files themselves.
Could we perhaps have a clean:postBuild task which, say, deletes something like build/src/**.js - these redundant files are very confusing in an IDE, since as well as having one copy of all the framework's source per project that depends on it, we are now likely to double this by each of those copies now having an extra copy within build/src - Cheers